generate_life_calendar.py

```diff
@@ -199,7 +200,10 @@ def draw_grid(ctx, date, birthdate, age, darken_until_date):
     for i in range(num_rows):
         # Generate string for current date
         ctx.set_source_rgb(0, 0, 0)
-        date_str = date.strftime('%d %b, %Y')
+        if classic:
+            date_str = '{: >3}'.format(i)
```

Comment:
Why would the date only be right-justified in "classic" mode? the original WBW calendar doesn't have dates

Reply:
I simply abused the variable to print the year, with padding (I am not sure how the text_extent() treats leading white space. I know it ignores trailing whitespace.)

Reply:
got it , my mistake, I thought the date was still being printed in "classic" mode. I didn't realize it was just printing a row number.